pkgs/games/vvvvvv/default.nix

```
@@ -0,0 +1,61 @@
+{ stdenv, fetchurl, fetchFromGitHub, requireFile
+, SDL2, SDL2_mixer, cmake, ninja
+, fullGame ? false }:
+
+let
+  dataZip = if fullGame then requireFile {
+    # the data file for the full game
+    name = "data.zip";
+    sha256 = "1q2pzscrglmwfgdl8yj300wymwskh51iq66l4xcd0qk0q3g3rbkg";
+    message = ''
+      In order to install VVVVVV, you must first download the game's
+      data file (data.zip) as it is not released freely.
+      Once you have downloaded the file, place it in your current
+      directory, use the following command and re-run the installation:
+      nix-prefetch-url file://\$PWD/data.zip
+    '';
+  } else fetchurl {
+    # the data file for the free Make and Play edition
+    url = https://thelettervsixtim.es/makeandplay/data.zip;
+    sha256 = "1q2pzscrglmwfgdl8yj300wymwskh51iq66l4xcd0qk0q3g3rbkg";
+  };
+
+  # if the user does not own the full game, build the Make and Play edition
+  flags = if fullGame then [] else [ "-DMAKEANDPLAY" ];
+in stdenv.mkDerivation rec {
+  pname = "vvvvvv";
+  version = "2.3-git-2020-01-22";
```

According to [Package naming](https://nixos.org/nixpkgs/manual/#sec-package-naming):
```suggestion
  version = "unstable-2020-01-22";
```